frontend/dialob-composer-material/vite.config.items.ts

```diff
@@ -0,0 +1,39 @@
+import { defineConfig } from 'vite';
+import path from 'path';
+import dts from 'vite-plugin-dts';
+
+export default defineConfig({
+  plugins: [dts({
+    include: 'src/items/**',  // Only include the folder being built
+  })],
+  build: {
+    outDir: 'src/items/dist', // Output to a dist folder within the folder being built
```

Comment:
I think it would be best if the output folder is outside the source tree (application is built to `/dist`, so library could be something like `/dist-lib` and it should be also added to `.gitignore` as it shouldn't be committed to git.